### ISSUE TITLE
There may be duplicates between the training data of BDD and the data of the validation set

### ISSUE THREAD

**Issue by FanGShiYuu** [sungyeonparkk/vision-assistant-for-driving#61]
Excellent work, especially the validation under multiple datasets. 

However, during the usage process, I found that for the BDD dataset, the train and val json files provided by the author seemed to have repetitive questions and answers. It's worth checking again.

<img width="1227" alt="Image" src="https://github.com/user-attachments/assets/e08ee903-df4d-4f25-a0da-8e423d4765e8" />
